docs/section10/10-14.md

```diff
@@ -0,0 +1,167 @@
+            - Else if, check that design cooling capacity does not equal null:`Elif cooling_system_b.design_total_cool_capacity != Null:`  
+                - Set cooling capacity equal to the design cooling capacity: `total_cool_capacity_b = cooling_system_b.design_total_cool_capacity`
+            - Else, set the was_capacity_not_defined boolean variable equal to TRUE: `Else: was_cool_capacity_not_defined = TRUE`  
+            - SYSTEM 4 <65000  
+            - Check if cooling capacity is <65,000 Btuh OR the zone aggregation factor was defined and the capacity is less than 65000 Btuh OR the capacity was not defined (this is one of the most stringent) OR the aggregation factor associated with the zone equals null: `if (zone_list_b[0].aggregation_factor != Null AND total_heating_capacity_b/zone_list_b[0].aggregation_factor < 65000) OR total_cooling_capacity_b < 65000 OR was_cool_capacity_not_defined == TRUE OR zone_list_b[0].aggregation_factor == Null:`  
+                - Set the expected baseline efficiency for <65000 (if the modeled part OR full load efficiency values equals 3.4 then it passes; this is captured in the subsequent logic): `expected_baseline_eff_b = 3.4`  
+                    - Loop through each of the efficiency_metric_types associated with the heating system until PART_LOAD_EFFICIENCY_NO_FAN  is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b. : `For x in range(len(heating_system_b.efficiency_metric_types)):`
+                        - Check if the efficiency metric equals PART_LOAD_EFFICIENCY_NO_FAN : `if heating_system_b.efficiency_metric_types[x] == "PART_LOAD_EFFICIENCY_NO_FAN":` 
+                            - Set the modeled efficiency value: `modeled_efficiency_b = heating_system_b.efficiency_metric_values[x]`  
+                        - Else, do nothing and move on to the next metric: `Else:`   
+                    - Check that modeled_efficiency_b does not equal Null: `if modeled_efficiency_b != Null:`    
+                        - Check if the modeled efficiency meets the expected baseline efficiency : `efficiency_matches_acceptable_value = expected_baseline_eff_b == modeled_efficiency_b`                
+                    - Check if efficiency_matches_acceptable_value equals False, if it equals True then skip below: `if efficiency_matches_acceptable_value == False:`  
+                        - Reset modeled_efficiency_b to equal Null: `modeled_efficiency_b = Null`  
+                        - Loop through each of the efficiency_metric_types associated with the heating system until FULL_LOAD_EFFICIENCY_NO_FAN is found. Get the associated value of the efficiency from the list of efficiency_metric_values associated with the heating_system_b : `For x in range(len(heating_system_b.efficiency_metric_types)):`
```

Comment:
`FULL_LOAD_EFFICIENCY_NO_FAN` should be `FULL_LOAD_COEFFICIENT_OF_PERFORMANCE_NO_FAN`